### PR TITLE
theme interface changes

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

* Added theme switch to modules/installation/pages/webui-setup.adoc
* Added theme default to modules/administration/pages/users.adoc
* Xreffed everything to everything else

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/uyuni-project/uyuni-docs/issues/614

### CODE REVIEW

File: modules/administration/pages/users.adoc [hunk 1]
```diff
@@ -153,3 +154,27 @@ Where a translation is not available, the {webui} will default to English (``en_
...
+
+
+
+=== User Default Interface Theme
+
+By default, the {productname} {webui} uses the theme appropriate to the product you have installed.
+You can change the theme to reflect the {uyuni} or {susemgr} colors.
+The {susemgr} theme also has a dark option available.
+
+After a user has been created, you can change the default theme in the ``rhn.conf`` configuration file.
```
Comment (keichwa) [inline modules/administration/pages/users.adoc:166]: this does not seem to depend on the created user.  it seems to be a system-wide setting.

**Review by jcayouette — APPROVED** [review]

File: modules/administration/pages/users.adoc (hunk 1 above)
Comment (Loquacity) [inline modules/administration/pages/users.adoc:166]:
```suggestion
You can change the default theme in the ``rhn.conf`` configuration file.
```

**Comment by ncounter** [comment]
It seems I came a bit late. Anyway, thank you for this PR :heart:

Question: I guess this detailed description will not go into 4.1, right?

**Comment by Loquacity** [comment]
> It seems I came a bit late. Anyway, thank you for this PR 

np. If you have further comments, let me know.

> 
> Question: I guess this detailed description will not go into 4.1, right?

It will go into 4.1.3, which is why I merged it so quickly, you slipped under the wire! ;)

**Pull request merged**